### PR TITLE
#1 comma separation

### PR DESCRIPTION
**Pull request by pentarex**
#1

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -36,21 +37,29 @@ describe('parse-mentions', function() {
...
       {name: 'doowb', mention: '@doowb', index: 2},
-      {name: 'jonschlinkert', mention: '@jonschlinkert', index: 11}
+      {name: 'jonschlinkert', mention: '@jonschlinkert', index: 11},
+      {name: 'pentarex', mention: '@pentarex', index: 29}
     ]);
   });
 
   it('should return string with the mentions replaced using the replace function', function() {
-    assert.equal(parse(input, replace('https://github.com')), '- [@doowb](https://github.com/doowb)\n- [@jonschlinkert](https://github.com/jonschlinkert)');
+    //TODO Not sure why the normal input is not working here, even in regular situation where there is a sentence like
```
Comment (jonschlinkert) [inline test.js:52]: try changing the regex to `/(\w@)|@([\w_]{0,15})(?=$|[\s\W])/g`

Comment (jonschlinkert) [reply test.js:52]: This should work, since we want to match up the end-of-string (`$`) or a whitespace (`\s`) or a non-word character (`\W`)

**Comment by pentarex** [comment]
Ok I will change it tomorrow. Thanks for the feedback

**Comment by doowb** [comment]
Thanks for the PR @pentarex! I'll get this published soon.

**Pull request merged**